modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
++
+.YAML manifest for the `ClusterLogForwarder` CR if you installed OpenShift Logging version 6
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: openshift-logging
+spec:
+  inputs:
+  - application:
+      selector:
+        matchLabels:
+          app.kubernetes.io/managed-by: tekton-pipelines
+    name: only-tekton
+    type: application
+  managementState: Managed
+  outputs:
+  - lokiStack:
+      labelKeys:
+        application:
+          ignoreGlobal: true
+          labelKeys:
+          - log_type
+          - kubernetes.namespace_name
+          - openshift_cluster_id
+      authentication:
+        token:
+          from: serviceAccount
+      target:
+        name: logging-loki
+        namespace: openshift-logging
+    name: default-lokistack
+    tls:
+      ca:
+        configMapName: openshift-service-ca.crt
+        key: service-ca.crt
+    type: lokiStack
+  pipelines:
+  - inputRefs:
+    - only-tekton
+    name: default-logstore
+    outputRefs:
+    - default-lokistack
+  serviceAccount:
+    name: collector
+----
++
+.YAML manifest for the `ClusterLogForwarder` CR if you installed OpenShift Logging version 5
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  inputs:
+  - name: only-tekton
+    application:
+      selector:
+        matchLabels:
+          app.kubernetes.io/managed-by: tekton-pipelines
+  pipelines:
+    - name: enable-default-log-store
+      inputRefs: [ only-tekton ]
+      outputRefs: [ default ]
+----
++
+Finally, in the `TektonResult` CR in the `openshift-pipelines` namespace, set the following additional parameters:
++
+--
+* `loki_stack_name`: The name of the `LokiStack` CR, typically `logging-loki`.
+* `loki_stack_namespace`: The name of the namespace where LokiStack is deployed, typically `openshift-logging`.
+--
++
+.Example of configuring LokiStack log forwarding in the `TektonResult` CR
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  targetNamespace: tekton-pipelines
+# ...
+  loki_stack_name: logging-loki
+  loki_stack_namespace: openshift-logging
+----
+
+[id="breaking-changes-1-16_{context}"]
+== Breaking changes
+
+* With this update, the metric name for the `EventListener` object for pipelines triggers that counts received events is changed from `eventlistener_event_count` to `eventlistener_event_received_count`.
+
+[id="known-issues-1-16_{context}"]
+== Known issues
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16 and later.
+
+[id="fixed-issues-1-16_{context}"]
+== Fixed issues
```

Comment:
I think based on SSG, Fixed issues should be Bug fixes: https://redhat-documentation.github.io/supplementary-style-guide/#release-notes. That's the category.

Reply:
Again, "Fixed issues" was used in previous Pipelines version RNs, so for consistency we need to keep it - pending a review of RN practices to harmonize them for all the layered products.